lib/main.dart

```diff
@@ -76,11 +52,31 @@ class _MyAppState extends State<MyApp> {
     );
   }
 
-  Scaffold _waitForAmplify() {
-    return const Scaffold(
-      body: Center(
-        child: CircularProgressIndicator(),
-      ),
-    );
+  void _configureAmplify() async {
```

Comment:
Technically, without us knowing, this returns a Future type of void. 
```suggestion
  Future<void> _configureAmplify() async {
```